tests/files/rel_auth.json

```diff
@@ -0,0 +1,59 @@
+{
+  "type": "dataset",
+  "id": "relauth",
+  "title": "Relations with auth",
+  "status": "beschikbaar",
+  "version": "1.2.3",
+  "tables": [
+    {
+      "id": "base",
+      "type": "table",
+      "version": "1.2.3",
+      "auth": ["SCOPE/HAMMERTIME"],
```

Comment:
I like the names :-)

Reply:
Finally, someone notices. Amsterdam/dso-api#417